### PR TITLE
 Feature/html content validation fixes

### PR DESCRIPTION
**Pull request by achen2401**
address this story:  https://www.pivotaltracker.com/story/show/143329407

- validate html against w3 validator to make sure HTML conforms to standards
- fix strayed tags
- fix invalid element attributes
- add alt text to images

### CODE REVIEW

File: portal/templates/gil/about.html [hunk 1]
```diff
@@ -76,22 +78,22 @@ <h4 class="icon-box__head">Care Plan</h4><span class="icon-box__lowhead">Navigat
...
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/colorado.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/DanaFarber.png') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/duke.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/emory.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/johnshopkins.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/karmanos.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/sloan.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/michigan.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/moffitt.gif') }}"></div>
+            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/colorado.gif') }}" alt="University of Colorado Cancer Center"></div>
```
Comment (pbugni) [inline portal/templates/gil/about.html]: translations?

File: portal/static/css/eproms.css [hunk 2]
```diff
@@ -3115,6 +3120,15 @@ div.or {
   margin: -1.3em auto 20px auto;
   width: 100%;
 }
+.password-popover { min-width: 200px; max-width: 250px; right: 50px !important;}
+.password-popover .popover-content { padding: 1.2em 1em 1.5em 1em; width: 100%; margin: 0 auto;}
+#pwdHintList {padding: 0; margin:2px; list-style-type: none}
```
Comment (pbugni) [inline portal/static/css/eproms.css]: intentional to have `#` on pwdHintList only?

File: portal/templates/gil/404.html [hunk 3]
```diff
@@ -1,4 +1,6 @@
 {% extends "gil/base.html" %}
+{% block title %}Truenth Page Not Found{% endblock %}
```
Comment (pbugni) [inline portal/templates/gil/404.html]: translation?

File: portal/templates/gil/500.html [hunk 4]
```diff
@@ -1,4 +1,6 @@
 {% extends "gil/base.html" %}
+{% block title %}Truenth Error{% endblock %}
```
Comment (pbugni) [inline portal/templates/gil/500.html]: translation?

File: portal/templates/gil/about.html (hunk 1 above)
Comment (achen2401) [inline portal/templates/gil/about.html]: yeah, Paul, you may have noticed I didn't put tranlation tags on GIL pages, I plan to do them en mass in a separate PR as most GIL pages don't have them just yet.

File: portal/static/css/eproms.css (hunk 2 above)
Comment (achen2401) [inline portal/static/css/eproms.css]: Paul, yes, the other ones reference css class name, hence the . notation

File: portal/templates/gil/500.html (hunk 4 above)
Comment (achen2401) [inline portal/templates/gil/500.html]: plan to do this in a separate PR for all GIL pages (see my above comment)
